<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_Settings</class>
 <widget class="QWidget" name="w_Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pb_Close">
     <property name="text">
      <string>Close</string>
     </property>
     <property name="icon">
      <iconset resource="../../Icons/Icons.qrc">
       <normaloff>:/General/Cancel.png</normaloff>:/General/Cancel.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pb_Save">
     <property name="text">
      <string>Save</string>
     </property>
     <property name="icon">
      <iconset resource="../../Icons/Icons.qrc">
       <normaloff>:/General/Save.png</normaloff>:/General/Save.png</iconset>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>685</width>
           <height>496</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Network</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="1">
              <widget class="QLineEdit" name="le_ServerAddress"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="lb_ServerAddress">
               <property name="text">
                <string>Server address:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="lb_ServerPort">
               <property name="text">
                <string>Server port:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="le_ServerPort"/>
             </item>
             <item row="2" column="0">
              <widget class="QCheckBox" name="cb_DevServer">
               <property name="text">
                <string>Use '127.0.0.1.4443'</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="tristate">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Icons/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
